app/src/main/java/com/mertceyhan/bitcoinmarket/features/market/domain/usecase/MarketInformationUseCaseImpl.kt

```diff
@@ -15,7 +14,7 @@ class MarketInformationUseCaseImpl @Inject constructor(
     private val marketModelMapper: MarketInformationMapper
 ) : MarketInformationUseCase {
 
-    override suspend fun getMarketInformation(timespan: MarketInformationTimespan): Flow<State<MarketInformation>> =
+    override suspend fun getMarketInformation(timespan: MarketInformationTimespan): Flow<com.mertceyhan.bitcoinmarket.base.data.State<MarketInformation>> =
```

Comment:
Could you add it as import to the top of the file?

`import com.mertceyhan.bitcoinmarket.base.data.State`